utils/notification_service.py

```diff
@@ -397,8 +397,12 @@ def error_out(title, ci_title="", runner_not_available=False, runner_failed=Fals
             ci_title_block = {"type": "section", "text": {"type": "mrkdwn", "text": ci_title}}
             blocks.append(ci_title_block)
 
+        offline_runners = []
         if runner_not_available:
             text = "💔 CI runners are not available! Tests are not run. 😭"
+            result = os.environ.get("OFFLINE_RUNNERS")
+            if result is not None:
```

Comment:
The env. variable `OFFLINE_RUNNERS` is only created in `check_runner_status.yml` workflow (so far)